l1-contracts/src/governance/Gerousia.sol

```diff
@@ -151,4 +150,8 @@ contract Gerousia is IGerousia {
   function computeRound(Slot _slot) public view override(IGerousia) returns (uint256) {
     return _slot.unwrap() / M;
   }
+
+  function getApella() public view override(IGerousia) returns (IApella) {
```

Comment:
Is this purely a gas optimization?

Reply:
No, it is more expensive in gas, it is there just to make the deployments easier as we otherwise want to have addresses for gerousia to deploy apella and vice versa making an annoying circular dependency.